components/Chips/src/MDCChipCollectionViewFlowLayout.m

```diff
@@ -31,7 +39,8 @@ @implementation MDCChipCollectionViewFlowLayout
       [NSMutableArray arrayWithCapacity:layoutAttributes.count];
 
   if (layoutAttributes.count > 0) {
-    UICollectionViewLayoutAttributes *attrs = layoutAttributes[0];
+    UICollectionViewLayoutAttributes *attrs = [layoutAttributes[0] copy];
+    attrs.frame = CGRectLeftAlign(attrs.frame);
```

Comment:
If we're laying-out the collection for RTL, do we need to swap the alignment? I would expect the order of chips to be reversed in the UI but the same in the underlying model.